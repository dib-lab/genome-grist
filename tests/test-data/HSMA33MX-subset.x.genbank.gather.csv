intersect_bp,f_orig_query,f_match,f_unique_to_query,f_unique_weighted,average_abund,median_abund,std_abund,name,filename,md5,f_match_orig
450000,0.4961411245865491,0.08710801393728224,0.4961411245865491,0.4817567567567568,1.5844444444444445,1.0,1.014660437027647,"GCA_001881345.1 Escherichia coli strain=SF-596, ASM188134v1",outputs/genbank/HSMA33MX-subset.x.genbank.prefetch.sig,683df1ec13872b4b98d59e98b355b52c,0.08710801393728224
357000,0.3936052921719956,0.0959935466523259,0.3936052921719956,0.40202702702702703,1.6666666666666667,1.0,1.0496542492413934,"GCA_009494275.1 Prevotella copri strain=iAK1214, ASM949427v1",outputs/genbank/HSMA33MX-subset.x.genbank.prefetch.sig,06cd82b5d11fdd59816052cc8bea4883,0.0959935466523259
