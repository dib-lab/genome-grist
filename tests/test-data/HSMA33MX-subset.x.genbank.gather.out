

overlap     p_query p_match avg_abund
---------   ------- ------- ---------
450.0 kbp     48.2%    8.7%       1.6    GCA_001881345.1 Escherichia coli stra...
357.0 kbp     40.2%    9.6%       1.7    GCA_009494275.1 Prevotella copri stra...

found 2 matches total;
the recovered matches hit 88.4% of the query

